### DISCUSSION TITLE
Password should change for each session.

### DISCUSSION THREAD

**Discussion by ackid32** [rustdesk/rustdesk#2123] [General]
The situation is :
I have some problem with my software, so the support team of the software want to connect my computer. They ask password and I gave it to them. Until I close the session, they can use the same password to connect (if the connection lost). They solved the problem, Now I have to change the password (so they cannot connect me again- without my knowledge). 

If the password change on each run, the problem will solved. This is one of the best feature in TeamViewer.

Is there any way?

**Comment by rustdesk** [comment]
in-depth discussion is here https://github.com/rustdesk/rustdesk/discussions/1869
